src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -1728,16 +1733,17 @@ public void registerFunction(
                 protected Void job(final SQLiteConnection sqliteConnection)
                     throws CatalogException, SQLiteException {
                   try {
-                    deleteFunctionIfExists(sqliteConnection, functionName, false);
+                    deleteFunctionIfExists(sqliteConnection, functionName);
```

Comment:
You mean I should replace with an ```UPDATE``` instead? If we don't delete the row I thought we would run into issues if we try to ```INSERT``` with the same key again